src/main/java/com/comphenix/protocol/wrappers/WrappedLevelChunkData.java

```diff
@@ -268,27 +275,63 @@ public List<byte[]> getBlockUpdates() {
 
         /**
          * Whether edges can be trusted for light updates or not.
-         *
+         * @deprecated Removed in 1.20
          * @return {@code true} if edges can be trusted, {@code false} otherwise.
          */
+        @Deprecated
         public boolean isTrustEdges() {
+            if(MinecraftVersion.TRAILS_AND_TAILS.atOrAbove()) {
+                return dummyTrustEdges; // ensure backwards compatability and prevent inconsistent states
+            }
             return (boolean) TRUST_EDGES_ACCESSOR.get(handle);
         }
 
         /**
          * Sets whether edges can be trusted for light updates or not.
-         *
+         * @deprecated Removed in 1.20
          * @param trustEdges the new value
          */
+        @Deprecated
         public void setTrustEdges(boolean trustEdges) {
+            if(MinecraftVersion.TRAILS_AND_TAILS.atOrAbove()) {
+                dummyTrustEdges = trustEdges; // ensure backwards compatability and prevent inconsistent states
+                return;
+            }
             TRUST_EDGES_ACCESSOR.set(handle, trustEdges);
         }
 
+        /**
+         * Constructs new LightData from values
+         * @param trustEdges Ignored
+         * @param skyYMask skyYMask
+         * @param blockYMask blockYMask
+         * @param emptySkyYMask emptySkyYMask
+         * @param emptyBlockYMask emptyBlockYMask
+         * @param skyUpdates skyUpdates
+         * @param blockUpdates blockUpdates
+         * @deprecated Parameter trustEdges was removed in 1.20
+         * @return new light data
+         */
+        @Deprecated
         public static LightData fromValues(BitSet skyYMask, BitSet blockYMask, BitSet emptySkyYMask, BitSet emptyBlockYMask,
                                            List<byte[]> skyUpdates, List<byte[]> blockUpdates, boolean trustEdges) {
+            return fromValues(skyYMask, blockYMask, emptySkyYMask, emptyBlockYMask, skyUpdates, blockUpdates);
```

Comment:
Doing this breaks compatibility with people that use trustEdges on <1.20. You either need to call the setter here or re-add the call below